src/app/divik-results/shared/divik.service.ts

```diff
@@ -41,13 +43,13 @@ export class DivikService extends Service {
   }
 
   get(preparationId: number, divikId: number, level: number): Observable<Heatmap> {
-    const queryUrl = `${this.getBaseUrl()}/divikResult/${preparationId}?divikId=${divikId}&level=${level}`;
+    const queryUrl = this.getBasePreparationUrl() + sprintf(apiUrl.divikResultUrl, preparationId, divikId, level);
```

Comment:
The ultimate goal of extraction of these urls to single place is to have single point for all changes involving migration of functionalities between workers / other hosts providing API. In current form, if only DiviK gets detached from visualization API, we still have to dig into the code - it's maintainability is low.

We can even agree on creating `getDivikUrl` which returns *the same value* as `getBasePreparationUrl` (for now!), but is maintainable from one central point. When there comes a need to change the machine responsible for such functionality, it will be in one place.